components/Card/Card.js

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { recipes } from "../lib/recipes";
+import "./Card";
+import Image from "next/image";
+import { ContainerStyle, ImageStyle } from "./Card.styled";
+
+export default function RecipesList() {
+  return (
+    <>
+      {recipes.map((recipe) => (
+        <ContainerStyle key={recipe.id}>
+          <Image
+            src={`/images/${recipe.picture}`}
+            width={60}
+            height={50}
+            alt={recipe.title}
+          />
+          <div className="ContainerStyle--text">
```

Comment:
In our sessions we learned to implement styling (when working with styled-components like this:
`<ContainerStyle 
...rest of the code
/>` 

meaning without className but I am not sure how this works with ContainerStyle--text